cmd/coap/main.go

```diff
@@ -94,15 +94,15 @@ func main() {
 		return
 	}
 
-	authClient, authHandler, err := auth.SetupAuthz(ctx, authConfig)
+	thingsClient, thingsHandler, err := auth.SetupThingsClient(ctx, authConfig)
 	if err != nil {
 		logger.Error(err.Error())
 		exitCode = 1
 		return
 	}
-	defer authHandler.Close()
+	defer thingsHandler.Close()
 
-	logger.Info("Successfully connected to things grpc server " + authHandler.Secure())
+	logger.Info("Successfully connected to things gRPC server " + thingsHandler.Secure())
 
```

Comment:
Since auth service gRPC clients have logging like `AuthService gRPC client Successfully connected to auth gRPC server`, I expect things service gRPC client also have same kind of logging.
```go
	logger.Info("ThingsService gRPC client Successfully connected to things gRPC server " + thingsHandler.Secure())
```